packages/notification-center/src/components/popover-notification-center/PopoverNotificationCenter.tsx

```diff
@@ -21,6 +22,9 @@ interface IPopoverNotificationCenterProps {
   actionsResultBlock?: (templateIdentifier: string, messageAction: IMessageAction) => JSX.Element;
   tabs?: ITab[];
   showUserPreferences?: boolean;
+  offset?: number;
```

Comment:
I chose to have the prop name as `offset` instead of `gutter` as Mantine has renamed it to offset in v5.0+, and having it as offset makes it easy for the users to use our future version without breaking change

Reply:
Makes sense!